### DISCUSSION TITLE
Display final beatmap statistics (BPM, star rating, difficulty, etc.) in mod select for visibility during customisation

### DISCUSSION THREAD

**Discussion by Shutakobein** [ppy/osu#18189] [Ideas]
Curently while you are changing beatmap speed with wind up/down mod you need to check beatmap info to see the final bpm. I think this change will add some qol into osu lazer. Also being able to type a bpm instead of using slider would be cool.

Edit: Same thing also applies to DT and NC.

**Comment by frenzibyte** [comment]
Somewhat relevant to https://github.com/ppy/osu/discussions/18172, potentially a duplicate.

EDIT: Aforementioned thread has been marked as duplicate of this for the second point.

**Reply by Theighlin** [reply]
It was kinda my bad there to include 2 things into 1 discussion. I think that can be marked as answered, as the first point has been addressed, and keep this one open for the second one.

**Comment by Rekunan** [comment]
Someone has made a PR for this #24642

**Reply by Givikap120** [reply]
This is different issue
but i want to make another PR that fixes that he suggested, because i need this too

**Reply by Rekunan** [reply]
Wait yeah I glossed over the other parts, my bad.
